contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -12,14 +12,13 @@ packages:
 - "bsdmainutils"
 - "curl"
 - "faketime"
-- "g++-8-multilib"
 - "libtool"
 - "pkg-config"
 # Host dependencies.
-- "g++-8"
-- "g++-8-arm-linux-gnueabihf"
-- "g++-8-aarch64-linux-gnu"
-- "g++-8-riscv64-linux-gnu"
+- "g++"
+- "g++-arm-linux-gnueabihf"
+- "g++-aarch64-linux-gnu"
+- "g++-riscv64-linux-gnu"
```

Comment:
Let's not remove the explicit `-8` to be more explicit...
We can use `update-alternatives` in the script to change the default gcc/g++ I think